tests/test_adapter.py

```diff
@@ -4,9 +4,10 @@
 from unittest.mock import Mock
 
 import torch
-from conftest import RunIf
 from lightning import Fabric
 
+from conftest import RunIf
```

Comment:
And then all these import reorders will be undone with automatic code fixes 😄.

Reply:
Uh maybe I ran ruff incorrectly? It's fine. Let the linters do their thing